### ISSUE TITLE
Update logstash 1.4.2 --configtest error dialog to submit bug report to github

### ISSUE THREAD

**Issue by naisanza** [elasticsearch/logstash#2324]
The current version of the compiled logstash binary points to https://logstash.jira.com/ in the error dialog. See below for output sample:

```
[root@elk kibana-3.1.2]# /opt/logstash/bin/logstash -f /etc/logstash/conf.d/logstash.conf
Using milestone 2 input plugin 'file'. This plugin should be stable, but if you see strange behavior, please let us know! For more information on plugin milestones, see http://logstash.net/docs/1.4.2-modified/plugin-milestones {:level=>:warn}
Using milestone 1 input plugin 'syslog'. This plugin should work, but would benefit from use by folks like you. Please let us know if you find bugs or have suggestions on how to improve this plugin.  For more information on plugin milestones, see http://logstash.net/docs/1.4.2-modified/plugin-milestones {:level=>:warn}
+---------------------------------------------------------+
| An unexpected error occurred. This is probably a bug.   |
| You can find help with this problem in a few places:    |
|                                                         |
| * chat: #logstash IRC channel on freenode irc.          |
|     IRC via the web: http://goo.gl/TI4Ro                |
| * email: logstash-users@googlegroups.com                |
| * bug system: https://logstash.jira.com/                |
|                                                         |
+---------------------------------------------------------+
The error reported is:
  Illegal pattern component: l
```

**Comment by jordansissel** [comment]
This whole error message (the one in the text-art box) was removed because it was not effective. Related commit: https://github.com/elasticsearch/logstash/commit/bded0888d3acd3ec56dfc8ad3e95a6e248f90ad7

As a result, I think we can close this.

**Comment by naisanza** [comment]
I... actually read it

Closed, dialog box pending removal per [bded088](https://github.com/elasticsearch/logstash/commit/bded0888d3acd3ec56dfc8ad3e95a6e248f90ad7)

**Comment by naisanza** [comment]
I... actually read it

Closed, dialog box pending removal per [bded088](https://github.com/elasticsearch/logstash/commit/bded0888d3acd3ec56dfc8ad3e95a6e248f90ad7)

**Comment by jordansissel** [comment]
@naisanza yeah :(

The removal was mostly a result of a my personal frustrations with folks ignoring it.

I thank you for reading it, though :) The box may return one day
